packages/vats/src/core/boot-psm.js

```diff
@@ -0,0 +1,223 @@
+// @ts-check
+import { Far } from '@endo/far';
+import {
+  installGovAndPSMContracts,
+  makeAnchorAsset,
+  startPSM,
+  PSM_MANIFEST,
+} from '@agoric/inter-protocol/src/proposals/startPSM.js';
+// TODO: factor startEconomicCommittee out of econ-behaviors.js
+import { startEconomicCommittee } from '@agoric/inter-protocol/src/proposals/econ-behaviors.js';
+import { ECON_COMMITTEE_MANIFEST } from '@agoric/inter-protocol/src/proposals/core-proposal.js';
+import { makeAgoricNamesAccess, makePromiseSpace } from './utils.js';
+import { Stable, Stake } from '../tokens.js';
+import {
+  addBankAssets,
+  buildZoe,
+  installBootContracts,
+  makeAddressNameHubs,
+  makeBoard,
+  makeVatsFromBundles,
+  mintInitialSupply,
+} from './basic-behaviors.js';
+import * as utils from './utils.js';
+import {
+  bridgeCoreEval,
+  makeBridgeManager,
+  makeChainStorage,
+  publishAgoricNames,
+  startTimerService,
+} from './chain-behaviors.js';
+import { CHAIN_BOOTSTRAP_MANIFEST } from './manifest.js';
+import {
+  startWalletFactory,
+  WALLET_FACTORY_MANIFEST,
+} from './startWalletFactory.js';
+
+/** @typedef {import('@agoric/inter-protocol/src/proposals/econ-behaviors.js').EconomyBootstrapSpace} EconomyBootstrapSpace */
+
+/**
+ * PSM and gov contracts are available as
+ * named swingset bundles only in
+ * decentral-psm-config.json
+ */
+const PSM_GOV_INSTALL_MANIFEST = {
+  [installGovAndPSMContracts.name]: {
+    vatPowers: { D: true },
+    devices: { vatAdmin: true },
+    consume: { zoe: 'zoe' },
+    installation: {
+      produce: {
+        contractGovernor: 'zoe',
+        committee: 'zoe',
+        binaryVoteCounter: 'zoe',
```

Comment:
If the strings are only for labelling the graph, can they all be 'gov' rather than 'zoe'?

Reply:
installations all live in the zoe vat, IIUC